src/patternfly/components/Accordion/accordion.scss

```diff
@@ -57,85 +56,109 @@
   background-color: var(--pf-c-accordion--BackgroundColor);
   box-shadow: var(--pf-c-accordion--BoxShadow);
 
-  .pf-c-accordion__toggle {
-    display: flex;
-    align-items: center;
-    justify-content: space-between;
-    width: 100%;
-    padding: var(--pf-c-accordion__toggle--PaddingTop) var(--pf-c-accordion__toggle--PaddingRight) var(--pf-c-accordion__toggle--PaddingBottom) var(--pf-c-accordion__toggle--PaddingLeft);
-    border: 0;
-    border-left: var(--pf-c-accordion__toggle--m-expanded--BorderWidth) solid var(--pf-c-accordion__toggle--BorderLeftColor);
+  &.pf-m-no-box-shadow {
```

Comment:
Would you call this `pf-m-plain` since this modifier basically removes any border/shadow/line around the accordion?

Reply:
We could call it something else, but it's just removing the box-shadow at the moment. I don't know if there is anything else we plan on offering in a variation of the accordion without a box shadow.

@mcarrano what do you think, should we give this variation (no box shadow) a more meaningful name? You had also mentioned that you weren't sure why the accordion had a box shadow to begin with. One thing we could do (in a breaking change release) is remove the box shadow from the accordion entirely, and if you want a box shadow, you place it in a card. Though it's worth noting the default box shadow on the accordion is `--pf-global--BoxShadow--md`, and the box shadow on the card is `--pf-global--BoxShadow--sm`.

Reply:
I don't really have a good answer @mcoker .  @mceledonia do you know why we put a box shadow around the accordion? I agree that without the shadow, the accordion could be placed in a card to get this effect.

Reply:
I believe the accordion had the shadow as a stylistic effect for a particular use case where it existed above other content similar to a popover. 

As for using a card to achieve the same effect, the default card shadow is quite different than the one I see on the accordion (the card's shadow is smaller/tighter), so I would say as long as the card is flexible enough to utilize whatever type of shadow is needed, that should work. 

It makes sense to offer a version without the shadow for usage within a broader context, Kyle worked on this and is out for a while so I will investigate further as I believe this was in collaboration with the digital design team.

Reply:
> Though it's worth noting the default box shadow on the accordion is `--pf-global--BoxShadow--md`, and the box shadow on the card is `--pf-global--BoxShadow--sm`.

Woops, missed that! Yeah we would want to be sure we cover that as I have a feeling the box shadow was added to cover a use case where this existed above other content, but I can't say for sure. Going to investigate that like I said - but I think offering a modifier to remove it makes sense.

As for naming, I expect this non-shadow version to be used in a larger context like a form or some kind of content layout. I expect the shadow version to be used more or less on it's own, possibly as a result of an action to trigger it to appear. Maybe that can give some ideas on what to name it.

Reply:
Sorry for the rapid response, but I am thinking if you need the larger shadow for this, you might be able to utilize a popover to get the same effect. Card would potentially work as well like you previously mentioned. In that case, we could remove that shadow altogether.